src/states/adult.rs

```diff
@@ -285,10 +276,9 @@ impl Base for Adult {
         }
 
         if let Some(routing_msg) = self.filter_hop_message(msg)? {
-            self.dispatch_routing_message(routing_msg, outbox)
-        } else {
-            Ok(Transition::Stay)
-        }
+            self.dispatch_routing_message(routing_msg, outbox)?;
+        } 
+        Ok(Transition::Stay)        
```

Comment:
Is it better form?

Reply:
`dispatch_routing_message` now returns `OK(())` (adult.rs line 151), so it's necessary